CODEOWNERS

```diff
@@ -30,8 +30,8 @@
 /src/lib/ @MinaProtocol/protocol-eng-reviewers
 
 /src/lib/blake2/ @MinaProtocol/crypto-eng-reviewers
-/src/lib/cli_lib/ @MinaProtocol/product-eng-reviewers
-/src/lib/mina_commands @MinaProtocol/product-eng-reviewers
+/src/lib/cli_lib/ @MinaProtocol/protocol-eng-reviewers
+/src/lib/mina_commands @MinaProtocol/protocol-eng-reviewers
```

Comment:
I think we can remove these lines given protocol-eng owns `src/lib/` by default

Reply:
ah right!